dozer-ingestion/src/connectors/mysql/binlog.rs

```diff
@@ -141,7 +141,14 @@ impl BinlogIngestor<'_, '_, '_, '_, '_> {
         'binlog_read: while let Some(event) = self.binlog_stream.as_mut().unwrap().next().await {
             let binlog_event = event.map_err(MySQLConnectorError::BinlogReadError)?;
 
-            self.next_position.position = binlog_event.header().log_pos().into();
+            let is_artificial = binlog_event
```

Comment:
Should the binlog still be rotated on line 180 if the rotate event is artificial? If not, just doing `if is_artificial { continue; }` might more clearly signal the intent that this event is just skipped.